tdm-reader/nitdmreader.yml

```diff
@@ -134,39 +134,39 @@ definitions:
     properties:
       decimatex:
         type: integer
-        description: Number of values on the x-axis. This determines the number of points that can be displayed.
+        description: Number of values on the x axis. This determines the number of points that can be displayed.
```

Comment:
Are we removing the '-' for consistency w/ other docs? I thought x-axis was more correct.